velox/common/memory/Memory.cpp

```diff
@@ -63,18 +63,46 @@ std::shared_ptr<MemoryAllocator> createAllocator(
 std::unique_ptr<MemoryArbitrator> createArbitrator(
     const MemoryManagerOptions& options) {
   // TODO: consider to reserve a small amount of memory to compensate for the
-  // non-reclaimable cache memory which are pinned by query accesses if enabled.
-  return MemoryArbitrator::create(
-      {.kind = options.arbitratorKind,
-       .capacity =
-           std::min(options.arbitratorCapacity, options.allocatorCapacity),
-       .reservedCapacity = options.arbitratorReservedCapacity,
-       .memoryPoolReservedCapacity = options.memoryPoolReservedCapacity,
-       .memoryPoolTransferCapacity = options.memoryPoolTransferCapacity,
-       .memoryReclaimWaitMs = options.memoryReclaimWaitMs,
-       .globalArbitrationEnabled = options.globalArbitrationEnabled,
-       .arbitrationStateCheckCb = options.arbitrationStateCheckCb,
-       .checkUsageLeak = options.checkUsageLeak});
+  //  non-reclaimable cache memory which are pinned by query accesses if
```

Comment:
Since this is implementation specific, we shall leave the default settings in specific arbitrator backend?

Reply:
I don't quite get it. Could you please help to clarify? Do you mean if block or else block?